src/App/Enumeration.cpp

```diff
@@ -23,8 +23,8 @@
 #include "PreCompiled.h"
 #ifndef _PreComp_
 # include <cassert>
-# include <cstring>
-# include <cstdlib>
+# include <string>
```

Comment:
No need to include string and vector here. It's already included in the header file. cstring and string are totally different things. The former is to offer all C like string-related functions and the later about the std::string class